### PR TITLE
Improve environment flag

### PR DESCRIPTION
**Pull request by ryhinchey**
Hi there! I've been using this project lately and decided to start contributing.  This PR is in reference to issue https://github.com/pa11y/pa11y/issues/483.

### CODE REVIEW

File: bin/pa11y.js [hunk 1]
```diff
@@ -298,22 +300,14 @@ function collectOptions(val, array) {
...
-	} catch (error) {}
-
-	console.log(`Pa11y:      ${versions.pa11y}`);
-	console.log(`Node.js:    ${versions.node}`);
-	console.log(`npm:        ${versions.npm}`);
-	console.log(`OS:         ${versions.os} (${process.platform})`);
+	return envinfo.run({
+		System: ['OS', 'CPU', 'Memory'],
+		Binaries: ['Node', 'Yarn', 'npm'],
+		Browsers: ['Chrome', 'Firefox', 'Safari'],
```
Comment (joeyciechanowicz) [inline bin/pa11y.js]: We don't need the `Browsers` section as chromium is provided by `puppeteer`.

File: bin/pa11y.js [hunk 2]
```diff
@@ -298,22 +300,14 @@ function collectOptions(val, array) {
...
+		npmPackages: ['pa11y', 'pa11y-ci', 'puppeteer', 'pa11y-runner-axe']
```
Comment (joeyciechanowicz) [inline bin/pa11y.js]: This should just be `pa11y` IMO. The others can be found easily enough by checking the version on GitHub

File: bin/pa11y.js [hunk 3]
```diff
@@ -298,22 +300,14 @@ function collectOptions(val, array) {
...
-	try {
-		versions.npm = require('child_process').execSync('npm -v').toString().trim();
-	} catch (error) {}
-
-	console.log(`Pa11y:      ${versions.pa11y}`);
-	console.log(`Node.js:    ${versions.node}`);
-	console.log(`npm:        ${versions.npm}`);
-	console.log(`OS:         ${versions.os} (${process.platform})`);
+	return envinfo.run({
+		System: ['OS', 'CPU', 'Memory'],
```
Comment (joeyciechanowicz) [inline bin/pa11y.js]: `'Shell'` would probably be useful.

```suggestion
		System: ['OS', 'CPU', 'Memory', 'Shell'],
```

File: bin/pa11y.js [hunk 4]
```diff
@@ -117,7 +118,8 @@ function handleHelp() {
  */
 async function runProgram() {
 	if (program.environment) {
-		outputEnvironmentInfo();
+		const info = await outputEnvironmentInfo();
```
Comment (joeyciechanowicz) [inline bin/pa11y.js]: Can you add the `console.log` to `outputEnvironmentInfo` rather than having it in here.

**Comment by joeyciechanowicz** [comment]
Thanks so much for contributing! Good issue to pick up.

File: bin/pa11y.js (hunk 2 above)
Comment (ryhinchey) [reply bin/pa11y.js]: sounds good. if others are needed down the road they can be easily added

**Pull request merged**